.nextcloudignore

```diff
@@ -0,0 +1,38 @@
+.git
+.github
+.gitignore
+.tx
+.vscode
+.php-cs-fixer.*
+/.codecov.yml
+/.eslintrc.js
+/.gitattributes
+/.gitignore
+/.l10nignore
+/.nextcloudignore
+/.travis.yml
+/.pre-commit-config.yaml
+/babel.config.js
+/build
+/AUTHORS.md
+/CHANGELOG.md
+/CODE_OF_CONDUCT.md
+/README.md
+/SECURITY.md
+/composer.*
+/node_modules
+/screenshots
+/src
+/vendor
```

Comment:
This is dangerous IMO. Some repos don't use Mozart or any fancy thing to move dependencies in `lib/` so we might just wanna ignore `vendor/bin`.

Reply:
As we use almost the same file in all integrations, your notice is important.
Should  we change string from `/vendor` to `/vendor/bin`
if I understand correctly?

Reply:
Well if some Php packages are actually imported and used in the project, we should not exclude `vendor`.
So maybe yes, to be sure, let's never ignore `vendor` but always ignore `vendor/bin`.
As you wish:
* Just check this on future releases you make and adjust if needed
* Or change it everywhere if you feel like it :grin:

Reply:
changed here, in other let's change when something else will be added or adjusted to not create spam with one line of code 😁